eclass/virtualx.eclass

```diff
@@ -54,66 +53,19 @@ case ${VIRTUALX_REQUIRED} in
 	manual)
 		;;
 	always)
-		if [[ ${EAPI:-0} != [0123456] ]]; then
-			BDEPEND="${VIRTUALX_DEPEND}"
-		else
-			DEPEND="${VIRTUALX_DEPEND}"
-		fi
+		BDEPEND="${VIRTUALX_DEPEND}"
 		RDEPEND=""
 		;;
-	optional|tests)
-		[[ ${EAPI} == [45] ]] \
-			|| die 'Values "optional" and "tests" for VIRTUALX_REQUIRED are banned in EAPI > 5'
-		# deprecated section YAY.
-		eqawarn "VIRTUALX_REQUIRED=optional and VIRTUALX_REQUIRED=tests are deprecated."
-		eqawarn "You can drop the variable definition completely from ebuild,"
-		eqawarn "because it is default behaviour."
-
-		if [[ -n ${VIRTUALX_USE} ]]; then
-			# so they like to specify the useflag
-			eqawarn "VIRTUALX_USE variable is deprecated."
-			eqawarn "Please read eclass manpage to find out how to use VIRTUALX_REQUIRED"
-			eqawarn "to achieve the same behaviour."
-		fi
-
-		[[ -z ${VIRTUALX_USE} ]] && VIRTUALX_USE="test"
-		DEPEND="${VIRTUALX_USE}? ( ${VIRTUALX_DEPEND} )"
-		RDEPEND=""
-		IUSE="${VIRTUALX_USE}"
-		;;
 	*)
-		if [[ ${EAPI:-0} != [0123456] ]]; then
-			BDEPEND="${VIRTUALX_REQUIRED}? ( ${VIRTUALX_DEPEND} )"
-		else
-			DEPEND="${VIRTUALX_REQUIRED}? ( ${VIRTUALX_DEPEND} )"
-		fi
+		BDEPEND="${VIRTUALX_REQUIRED}? ( ${VIRTUALX_DEPEND} )"
 		RDEPEND=""
 		IUSE="${VIRTUALX_REQUIRED}"
 		[[ ${VIRTUALX_REQUIRED} == test ]] &&
 			RESTRICT+=" !test? ( test )"
 		;;
 esac
 
-# @FUNCTION: virtualmake
-# @DESCRIPTION:
-# Function which start new Xvfb session
-# where the VIRTUALX_COMMAND variable content gets executed.
-virtualmake() {
```

Comment:
`virtualmake` seems to be used in `Xemake` (which has users) and in two packages: net-nds/jxplorer and dev-python/pmw.

Reply:
kept them as stub functions that unconditionally die.